geoportal/tests/test_get_url.py

```diff
@@ -54,7 +54,7 @@ def static_url(path, **kwargs):
         request.static_url = static_url
 
         self.assertEqual(get_url2("test", "static://pr:st/icon.png", request, set()), "http://server.org/pr:st/icon.png")
-        self.assertEqual(get_url2("test", "static:///icon.png", request, set()), "http://server.org/my_project_geoportal:static/icon.png")
+        self.assertEqual(get_url2("test", "static:///icon.png", request, set()), "http://server.org//etc/geomapfish/static/icon.png")
```

Comment:
```suggestion
        self.assertEqual(get_url2("test", "static:///icon.png", request, set()), "http://server.org/etc/geomapfish/static/icon.png")
```
double slash in the url ?

Reply:
It's just an artifact of the tests...

Reply:
It should be like that or also changing the get static URL method ...